contracts/pol/interfaces/ICarbonPOL.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { IUpgradeable } from "../../utility/interfaces/IUpgradeable.sol";
+import { Token } from "../../token/Token.sol";
+
+/**
+ * @dev CarbonPOL interface
+ */
+interface ICarbonPOL is IUpgradeable {
+    error DuplicateToken();
+    error InvalidToken();
+    error InvalidTokenLength();
+    error TokenPoolFeesLengthMismatch();
+
+    /**
+     * @dev triggered after a successful trade is executed
```

Comment:
Nitpick: Eventually, we should replace theses @devs with @notice when applicable

Reply:
replaced